src/app/store.ts

```diff
@@ -1,10 +1,21 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+// eslint-disable-next-line import/extensions
+import usersReducer from '../features/users';
```

Comment:
The import statement for `usersReducer` lacks a file extension. Ensure that your project's module resolution settings support this, or consider adding the appropriate file extension to avoid potential import issues.